doc/README.md

```diff
@@ -101,8 +100,6 @@
 
 The recommended approach is to deploy APISIX with [AWS CDK](https://aws.amazon.com/cdk/) on [AWS Fargate](https://aws.amazon.com/fargate/) which helps you decouple the APISIX layer and the upstream layer on top of a fully-managed and secure serverless container compute environment with autoscaling capabilities.
 
-See [this guide](https://github.com/pahud/cdk-samples/blob/master/typescript/apisix/README.md) by [Pahud Hsieh](https://github.com/pahud) and learn how to provision the recommended architecture 100% in AWS CDK.
```

Comment:
I have no idea what's the correct link, so I have to remove this paragraph.

cc @moonming @pahud

Reply:
No problem. Let's remove it.

Reply:
In the long term, it would be great to provide a `cdk-apisix` CDK construct library and user experience would be like

```ts
import * as apisix from 'cdk-apisix';

const workload = new apisix.Workload();
```

And everything a standard APISIX workload on AWS will be immediately provisioned.

However, at this moment, I personally prefer we ship the document as is and remove unnecessary paragraphs.

Reply:
ICYMI I have a video talking about what is `Contruct` in Mandarin and I compared it with the `Construct` concept in `The Matrix`. I believe you folks will love it.

https://www.youtube.com/watch?v=iVNs1NKWqX0&t=793s (13:13")